_config.yml

```diff
@@ -29,7 +29,7 @@ footer-links:
   twitter: jekyllrb
   stackoverflow: # your stackoverflow profile, e.g. "users/50476/bart-kiers"
 
-# Your disqus shortname, entering this will enable commenting on posts
```

Comment:
Great feedback, @krris. Shall we make the message even more explicit? How about...

```Your website's "shortname' defined on Disqus (not your Disqus username). Entering this will enable commenting on posts.```